README.md

```diff
@@ -0,0 +1,15 @@
+# LobbyHeads - 🌟 Decorative Heads for Your Minecraft Server Hub!
+
+LobbyHeads is a Minecraft server hub plugin for adorning your hub with decorative heads at any location. It permits
+designated players to replace these heads with personal ones, casting a hologram of their nickname and skin. The feature
+can be awarded to premium ranks as a special recognition.
+
```

Comment:
## Permisions

Plugin comes with two available permissions:
| Permision | Usage |
| :--- | :--- |
| lobbyheads.replace | Allows the User to replace heads on demand. |
| lobbyheads.admin | Allows the User adding or removing heads |